public/css/modules/videoBackground.css

```diff
@@ -0,0 +1,24 @@
+.fullscreen-bg {
```

Comment:
You don't need to set it to be display:none when not in use?

I suppose because it has no content, it just silently renders as transparent nothingness behind the page...

Reply:
Yes, exactly. I like to think of the page as being four pieces of stacked glass: the first shows the player and other clickable page elements, the second is the video display that is loaded and unloaded, the third is the `background-image` , and the fourth is `background` color.

Reply:
That is even a renderer-accurate view of the page... So fair enough.